### ISSUE TITLE
PTN008090861 (PTHR42909)

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#5079]
* **PTHR ID & PTN node:**
PTN008090861 (PTHR42909)

* **Sequences with problematic annotation (ID + gene/protein name):**
Dmel CG2794 (FBgn0031265/Q9VPR3) and orthologs.

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**
Erroneous source

Source annotations are:
This node is annotated with two very different MFs from 2 sources:
hydrolase activity, acting on glycosyl bonds IBD
pseudouridylate synthase activity IBD
PTN000062125 (ARATH\|TAIR=locus=2008081\|UniProtKB=Q84K35)
PTN000062134 (ECOLI\|EnsemblGenome=BAE76642\|UniProtKB=P33025)

For Q84K35 (aka PUMY), there's an IDA annotation to "GO:0030597 RNA glycosylase activity" from PMID:31907295.
That abstract says "ΨMP glycosylase (PUMY) that hydrolyzes ΨMP (pseudouridine monophosphate) to uracil and ribose-5-phosphate."
So I think that activity should be annotated with "pseudouridylate synthase activity GO:0004730" (def: Catalysis of the reaction: D-ribose 5-phosphate + uracil = H2O + pseudouridine 5'-phosphate." rather than RNA glycosylase activity GO:0030597 (def: Catalysis of the hydrolysis of N-glycosidic bonds in an RNA molecule.)
@tberardini 

For P33025, there are IDA annotations to "GO:0004730	pseudouridylate synthase activity" from PMID:23066817, and to "GO:0016798 hydrolase activity, acting on glycosyl bonds" from PMID:18591240.
The PMID:18591240 abstract says "The second enzyme, encoded by the gene yeiN belonging to the same operon as yeiC, catalyzes the conversion of pseudouridine 5'-phosphate to uracil and ribose 5-phosphate". So I think that should also be annotated with "pseudouridylate synthase activity GO:0004730" rather than "hydrolase activity, acting on glycosyl bonds".
(Not sure who to tag about correctly the source annotation)

Bottom line - I think the MF annotation on PTN008090861 should only be to "pseudouridylate synthase activity"

See #3608 for related InterPro2GO ticket.

**Comment by marcfeuermann** [comment]
Annotation reviewed
